.github/workflows/cd.yml

```diff
@@ -618,10 +618,16 @@ jobs:
       - name: Prepare parameters
         id: argo-parameters
         run: |
+          # Add prod-canary when prod is present in the environment
+          modified_environment="${ENVIRONMENT}"
+          if [[ "${ENVIRONMENT}" == *"prod"* && "${ENVIRONMENT}" != *"prod-canary"* ]]; then
```

Comment:
@xnyo for now, should we just trigger argo-workflow when prod is present in the environment so it updates the prod-canary `_catalog_version`.